### PR TITLE
Initial apache site

### PR DESCRIPTION
**Pull request by klucar**

### CODE REVIEW

File: incubator-site/_site/community/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/apache/incubator-myriad/pulls">Submit a patch:</a> Use Github for pull requests.</td>
+    </tr>
+  </tbody>
+</table>
+
+<h4 id="hangout-with-developers">Hangout with Developers</h4>
+
+<p>Apache Myriad developers gather bi-weekly for a Google Hangout to sync up commits, talk about features, upcoming events, and more. Subscribe to the dev mailing list to find out when and receive links to join before it starts.</p>
```
Comment (adam-mesos) [inline incubator-site/_site/community/index.html]: Can you mention that we meet bi-weekly "on Wednesdays" and link to the Meeting minutes doc: https://docs.google.com/a/mesosphere.io/document/d/1JGmJrgeg98bHw_0_sSRmyX6WiAe13OdErcFlaz6Aa04/edit#heading=h.aaws1n2fsgsi

**Comment by adam-mesos** [comment]
I've been browsing around the site and it looks awesome. Super polished and full of useful information/links. The biweekly hangout minutes is the only piece I found missing. Let's work with @kensipe to get this committed to the subversion repo and published.

**Comment by smarella** [comment]
@klucar 

@klucar Can you please add a ["Disclaimer"](http://incubator.apache.org/guides/branding.html#disclaimers)  as mandated by Apache for incubator projects? 

I browsed around the websites of some incubator projects and I liked how [Kylin](http://kylin.incubator.apache.org/development/) did it - they have the disclaimer in the footer for all the pages.

**Comment by klucar** [comment]
Site updated. I just took the text from the Kylin project and changed Kylin
to Myriad. I'm now ready for law school.

Preview here:
http://klucar.github.io/myriad-site/

**Comment by adam-mesos** [comment]
Please explain why this PR was closed. Because this was merged already, or because we'll just apply the site to the svn repo?
Maybe we should host the website source in git as well, so we can more easily make PRs, and then we'll just periodically update svn from the latest git site. We just have to be clear that all changes are made in git, and mirrored to svn, so manual changes to svn don't get overwritten.

**Comment by klucar** [comment]
It was my understanding from the last telecon that I would close this PR
because it was going to live in SVN. I'm for whatever is easiest. Ken and I
have been trying to hook up today to figure this out. Maybe let us play
with the SVN setup and we'll know which way will work better?

**Comment by smarella** [comment]
I feel it'll be easier to do PRs if the website source is in git (in
addition to being in SVN).

Santosh

**Comment by adam-mesos** [comment]
Sounds good, and it's what I thought. I just want to get the reason/discussion down in a comment here so anybody else passing by knows why.
Old Apache motto: "If it didn't happen on a mailing list, it didn't happen."

**Pull request closed without merging**